docs/issuer/on-chain-issuer/on-chain-tutorial.md

```diff
@@ -12,44 +12,42 @@ keywords:
   - smart contract
 ---
 
-This guide presents an example of how to use the **on-chain issuer**. We created a small application, where we communicate with Metamask to retrieve the user's balance and a claim about this balance is generated via the on-chain issuer.
+Currently, we have two approaches to issuing on-chain credentials: **on-chain merklized issuer** and **on-chain non-merklized issuer**.
+The difference between these two approaches:
+1. The **on-chain merklized issuer** calculates the Merkle root for a credential on the backend and stores the core claim of the credential on-chain using a smart contract. We can think of it as a centralized issuer with the ability to store trees on the blockchain.
+    - [demo](https://github.com/0xPolygonID/onchain-merklized-issuer-demo)
+    - [contract](https://github.com/0xPolygonID/contracts/blob/main/contracts/examples/IdentityExample.sol)
 
-There are three main components in this application:
+1. The **on-chain non-merklized issuer** can use information from the blockchain (such as balance, token ownership, etc.) to issue a credential directly on the blockchain. This approach is more decentralized, but must use only public data and has a limit on the number of fields in the credential. More about [**non-merklized credentials**](https://docs.iden3.io/protocol/non-merklized/).
```

Comment:
~~This approach is more decentralized, but must use only public data and has a limit on the number of fields in the credential.~~
This approach is decentralized and trustless - no need to trust an issuer to act honestly, because it's enforced by the smart contract and auditable on chain. But it comes with a few limitations: max 4 data fields in the credential and data is public.